pkg/worker/worker.go

```diff
@@ -51,7 +51,6 @@ func New(c Config) *Worker {
 		cou[MetricTotal] = recorder.NewCounter(recorder.CounterConfig{
 			Des: "the total amount of worker handler executions",
 			Lab: map[string][]string{
-				"service": {"specta"},
```

Comment:
Kinda unrelated to the frontend SLO. Just cleaning this up. We already track the origin of the emitted metrics, so we don't need to add this service label ourselves.